src/maybe.js.flow

```diff
@@ -2,7 +2,7 @@
 
 export type Maybe<A> = Just<A> | Nothing
 
-type AnyVal = number | boolean | string | Object
+type AnyVal = number | boolean | string | Object | Array<*>
```

Comment:
Actually this should work anyway because `maybe([1])` doesn't show errors. Array is an Object. Are you just trying to keep the inferred value of Array?